test-result-summary-client/src/utils/Utils.js

```diff
@@ -10,4 +10,48 @@ export const getInfoFromBuildName = (buildName) => {
         return {jdkVersion, jdkImpl, level, group, platform};
     }
     return null;
+}
+
+export const getGitDiffLinks = (before, after, buildName) => {

```

Comment:
There is no need to have three functions. `getGitDiffLinks()` can call `getSHA ()`.

To simplify the logic, we can have a map:

```
// set jckDiffLink

const diffLinks = [
"OpenJ9": "https://github.com/eclipse/openj9/compare/",
"OMR": "https://github.com/eclipse/omr/compare/",
"IBM": "<IBM_REPO>/compare/",
"JCL": jckDiffLink, 
]

// in this way, we can loop diffLinks and return obj.
```